src/Glpi/Form/QuestionType/AbstractQuestionTypeSelectable.php

```diff
@@ -363,6 +375,7 @@ class="form-check-input {{ hide_default_value_input ? 'd-none' : '' }}"
                     aria-label="{{ translations.default_option }}"
                     {{ checked ? 'checked' : '' }}
                     {{ disabled ? 'disabled' : '' }}
+                    {{ extra_input_attributes|raw }}
```

Comment:
`extra_input_attributes` should be passed as an array, so we can escape key/values properly in the template.

```suggestion
                    {{ extra_input_attributes|map(value, key) => key|e ~ '=" ~ value|e ~ '"')|join(' ')|raw }}
```